serverless/pages/pricing.asciidoc

```diff
@@ -6,51 +6,48 @@
 
 preview:[]
 
-Elasticsearch is priced based on the consumption of the underlying
-infrastructure used to support your use case, with the performance
-characteristics you need. We measure by Virtual Compute Units (VCUs), which is a
-slice of RAM, CPU and local disk for caching. The number of VCUs required will
-depend on the amount and the rate of data sent to Elasticsearch and retained,
-and the number of searches and latency you require for searches. In addition, if
-you required {ml} for inference or NLP tasks, those VCUs are also
+Elasticsearch is priced based on consumption of the underlying
+infrastructure that supports your use case, with the performance
+characteristics you need. Measurements are in Virtual Compute Units (VCUs).
+Each VCU represents a slice of RAM, CPU, and local disk for caching. 
+
+The number of VCUs required 
+depends on the amount and rate of data sent to Elasticsearch and retained,
+as well as the number of searches and the latency you require for searches. In addition, if
+you use {ml} for inference or NLP tasks, those VCUs are also
 metered and billed.
 
-include::../partials/minimum-vcus-detail.asciidoc[]
-
 [discrete]
 [[elasticsearch-billing-information-about-the-vcu-types-search-ingest-and-ml]]
-== Information about the VCU types (Search, Ingest, and ML)
+== VCU types: Search, Ingest, and ML
 
-There are three VCU types in Elasticsearch:
+Elasticsearch uses three VCU types:
 
-* **Indexing** — The VCUs used to index the incoming documents to be
-stored in Elasticsearch.
-* **Search** — The VCUs used to return search results with the latency and
-Queries per Second (QPS) you require.
-* **Machine Learning** — The VCUs used to perform inference, NLP tasks, and other ML activities.
+* **Indexing:** The VCUs used to index incoming documents.
+* **Search:** The VCUs used to return search results, with the latency and
+queries per second (QPS) you require.
+* **Machine learning:** The VCUs used to perform inference, NLP tasks, and other ML activities.
 
 [discrete]
 [[elasticsearch-billing-information-about-the-search-ai-lake-dimension-gb]]
-== Information about the Search AI Lake dimension (GB)
+== Search AI Lake dimension (GB)
 
-For Elasticsearch, the Search AI Lake is where data is stored and retained. This is
-charged in GBs for the size of data at rest. Depending on the enrichment,
-vectorization and other activities during ingest, this size may be different
+Elasticsearch stores and retains data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>, 
+which is charged in GB for the size of data at rest. Depending on enrichment,
+vectorization, and other activities during ingest, this size may be different
 from the original size of the source data.
```

Comment:
```suggestion
{es-serverless} projects store data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>. You are charged per GB of data at rest. Note that if you perform operations at ingest such as vectorization or enrichment, your storage size will differ from your original source data.
```

Suggestion to be more direct

Reply:
We might want an extra explicit statement here that says which of the 2 gets billed, or reword slightly the last sentence in the suggestion

Reply:
I just tweaked to say "stored data" in both sentences, is that clearer?